libs/backend/src/ui_strings/ui_strings_machine_configuration_test_runner.ts

```diff
@@ -153,7 +151,9 @@ export function runUiStringMachineConfigurationTests(
       audioIds: string[];
       languageCode: LanguageCode;
     }) {
-      return _.sortBy(store.getAudioClips(input), 'id');
+      return [...store.getAudioClips(input)].sort((a, b) =>
+        a.id.localeCompare(b.id)
+      );
```

Comment:
I saw that es2023 has [`Array::toSorted`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted), which will make this slightly nicer in the future.

Reply:
Woah that would be cool